benefit/data/sheets/AuthorisationCaseState/ AuthorisationCaseState-hmrc-nonprod.json

```diff
@@ -0,0 +1,8 @@
+[
+  {"LiveFrom": "16/07/2019", "CaseTypeID": "Benefit", "CaseStateID": "appealCreated", "AccessProfile": "caseworker-sscs-hmrcresponsewriter", "CRUD": "R"},
```

Comment:
Theres a space in the name of this file.

Reply:
Fixed.

Reply:
Still showing as having a space. Is that just a left over ?